packages/toast/src/components/ToastAlert/ToastAlert.tsx

```diff
@@ -1,9 +1,48 @@
-import { Alert, AlertProps } from "@gc-digital-talent/ui";
+import { ReactNode } from "react";
 
-const ToastAlert = ({ type, children }: AlertProps) => (
-  <Alert.Root live={false} type={type} className="m-0" tabIndex={0}>
-    <div className="leading-[1.1] xs:pr-12">{children}</div>
-  </Alert.Root>
+import { Notice, NoticeProps } from "@gc-digital-talent/ui";
+
+interface ToastMessageObject {
```

Comment:
I think the title is optional and the contents are mandatory?

<img width="837" height="183" alt="image" src="https://github.com/user-attachments/assets/2c05f8c3-d0d3-49d2-8b77-4bbafcf57d8a" />

Reply:
IK think that is another question for @JoshBeveridge . There was no guidelines for toasts. However, I did the inverse for toasts since the title is the only inidcation we have of the alert levlel (`error`, `warning`, `success`) which seemed important.

Reply:
SEE:

<img width="1047" height="452" alt="swappy-20251103_144050" src="https://github.com/user-attachments/assets/c6658086-967c-4e65-bd70-855a17004033" />